src/auth/auth.controller.ts

```diff
@@ -72,6 +76,46 @@ export class AuthController {
     this.clearCookie(res);
   }
 
+  @Get('kakao')
+  @UseGuards(KakaoGuard)
+  kakaoLogin() {
+    return;
+  }
+
+  @Get('kakao/callback')
+  @UseGuards(KakaoGuard)
```

Comment:
이렇게 해야 정상적으로 req.user에 값이 들어갈거같은데 혹시 테스트가 됐을까요??

WEB에서 리다이렉트를 시나리오할 수 있는 파일이 없어서 아마 테스트가 안됐을거같은데 AI를 이용해서 간단한 html을 생성해 테스트해보시는것도 좋을거같아요

```suggestion
  @UseGuards(AuthGuard('kakao'))
```

Reply:
`export class KakaoGuard extends AuthGuard('kakao')`로 선언해두었는데,`@UseGuards(AuthGuard('kakao'))`를 사용하는게 맞나요? 이것도 위에 처럼 http://localhost:3000/auth/kakao 로 테스트 할 때는 문제가 없었는데...